google/cloud/bigtable/testing/table_integration_test.cc

```diff
@@ -105,7 +110,43 @@ void TableIntegrationTest::SetUp() {
       TableTestEnvironment::project_id(), TableTestEnvironment::instance_id(),
       ClientOptions());
 
-  ASSERT_STATUS_OK(table_admin_->DropAllRows(TableTestEnvironment::table_id()));
+  // We cannot use `DropAllRows()` to cleanup the table because the integration
+  // tests against production sometimes use all the "'DropRowRangeGroup' and
```

Comment:
s/use/consume/ might be clearer.  I, at least, had to read the sentence twice.

Reply:
Fixed.